R/reverse_geocoding.R

```diff
@@ -77,6 +77,40 @@ oc_reverse <-
     oc_parse(res)
   }
 
+#' @export
+oc_reverse_df <-
+  function(latitude,
+           longitude,
+           key = oc_key(),
+           bounds = NULL,
+           countrycode = NULL,
+           language = NULL,
+           limit = 10,
```

Comment:
Maybe set the default to `limit = 1`. If you geocode multiple addresses in a dataframe you usually want to have one result per address, I think. Said differently, it may be confusing to have multiple results per source address.